lib/rufo/dot_file.rb

```diff
@@ -6,9 +6,7 @@ def initialize
   end
 
   def find_in(dir)
-    @cache.fetch(dir) do
-      @cache[dir] = internal_find_in(dir)
-    end
+    @cache[dir] ||= internal_find_in(dir)
```

Comment:
Just curious, what is the reason to change this behaviour?

This change might lead to more uncached lookups if a dot file was *not* found for `dir` (returning `nil`).

The previous implementation `@cach.fetch(dir)` was able to store `nil` values while the new impl cannot.

WDYT?